userguide/advanceduse/dav.rst

```diff
@@ -14,22 +14,48 @@ These instructions work for CalDAV as well.
 1) Follow this `guide <https://docs.ubports.com/en/latest/userguide/advanceduse/adb.html>`_ to activate Developer Mode and a connection either through ADB or SSH.
 
 
-2) Download this `script <https://gist.github.com/bastos77/0c47a94dd0bf3e394f879c0ff42b7839>`_ (let's call it dav.sh) and edit the following variables: 
+2) OPTION A: 
```

Comment:
Please use some expressive title, rather than "option A", maybe referring to the github user bastos77